### ISSUE TITLE
merit badge for contributors

### ISSUE THREAD

**Issue by mozfreddyb** [mozilla/OpenDesign#203]
## Goal: 
I want a design for a sticker that we can use as some sort of merit badge for people contributing to the security of Firefox (specifically the Firefox product).
This is meant to be a reward for people going out of their way to help the security team.

## Info: 
Stickers to be placed on alptops :)

## Style Information: 
It *might* be interesting exploring something along the lines of a [mission patch](https://en.wikipedia.org/wiki/Mission_patch), *maybe* even something that can be modified into various levels (like bronze/silver/gold).
This will be a tough balance to strike, but I also would like it *not* to look military.

## Deadline: 
Would be nice to have it in Q2

## Tag: 
[Design Needed]

**Comment by ruchikabgosain** [comment]
Hey @mozfreddyb do you still need this? Let me know, i'll be happy to take this up!

**Comment by pransh15** [comment]
Hey @ruchikabgosain, since you're willing to work on this, can you work on something generic - can be used for other teams as well?

Three Levels: Newbie, Master, Sensei? 👀
what do you think?

@kindavishal any inputs?

**Comment by mozfreddyb** [comment]
> Hey @mozfreddyb do you still need this? Let me know, i'll be happy to take this up!

Yes @ruchikabgosain, I'd still love to give out stickers as recognition. Would be great if you found something that allows being themed in various levels (bronze / silver / gold). Maybe the tagline "Defender Of The Fox" could be funny / inspirational? You decide :-)

Looking forward to seeing what you come up with!

**Comment by ruchikabgosain** [comment]
@mozfreddyb surely! let me get back to you with a rough sketch!

**Comment by kindavishal** [comment]
> Hey @ruchikabgosain, since you're willing to work on this, can you work on something generic - can be used for other teams as well?
> 
> Three Levels: Newbie, Master, Sensei? eyes
> what do you think?
> 
> @kindavishal any inputs?

Heyyy! Sorry for the super delayed reply!
Three level sounds good to me! If we are talking about generics, what groups are we considering?

**Comment by ruchikabgosain** [comment]
@kindavishal great then! i ll be considering 3 levels. Also, @mozfreddyb can you confirm to @kindavishal what levels are we talking about?

**Comment by mozfreddyb** [comment]
So, this badge is supposed to be "awarded" to everyone who contributes to the security of Firefox products. This could be finding a problem, fixing a security bug, etc.

If we (the Firefox security team) had three variants of the award badge, we could hand out stickers with bronze badges to a lot of people and get them interested in helping us more ;-) Silver badges are for great continuous contributions and so on.

Does that answer your question?

**Comment by kindavishal** [comment]
That is a nice idea, I really like it. Also, may I suggest to make them pins instead of badges?

**Comment by mozfreddyb** [comment]
To clarify, they should be stickers but they should *look* like badges. I want them to be stickers, because they have a high level of exposure and visibility (size, laptops are present at every meeting, etc.) and relatively cheap.

**Comment by kindavishal** [comment]
Okkk! Got it!
@pransh15 @ruchikabgosain I think we now have a clear idea what is needed here! What say?

**Comment by pransh15** [comment]
Sure, I think it's pretty clear.
Can you both take this one forward?

**Comment by likeanushkaa** [comment]
Is this issue still open?

**Comment by mozfreddyb** [comment]
Sure, nobody ever designed a sticker here.
Please read through the full thread and once you have a draft, please create a new pull request with your suggestion for review & design iteration. Feel free to tag me for review, not sure who else in this thread is still actively participating in open design. :)

**Comment by ozgeoztnr** [comment]
Hello,

I designed 3 badges: Bronze, Silver, Gold. I'm not sure if you need more complex designs.
I would love to hear your opinion @mozfreddyb 

![Badges(1)](https://user-images.githubusercontent.com/84623104/119268464-ccace380-bbfb-11eb-93ab-1f5f193d5f2d.png)

<img width="658" alt="Bronze" src="https://user-images.githubusercontent.com/84623104/119268490-f2d28380-bbfb-11eb-8980-b1f5d990c931.png">
<img width="658" alt="Gold" src="https://user-images.githubusercontent.com/84623104/119268491-f403b080-bbfb-11eb-8d59-6c2fb8ac4f30.png">
<img width="658" alt="Silver" src="https://user-images.githubusercontent.com/84623104/119268496-f5cd7400-bbfb-11eb-9060-aefdd45248fc.png">

**Comment by mikekellyonline** [comment]
Have these designs been approved? Also, are they being used? If not, I would be interested in creating more designs.

**Comment by Kuberwastaken** [comment]
Following up on this thread, would love to design some cool stickers for this
